website/static/js/logFeed.js

```diff
@@ -83,9 +83,13 @@ var Log = function(params) {
         return ret;
     });
 
-    //helper function to strip the leading slash for file or folder in log template
-    self.stripLeadingSlash = function(path){
-        return path.replace(/^\//, '');
+    //helper function to strip the slash for file or folder in log template
+    self.stripSlash = function(path, name){
```

Comment:
That being said, if we make an assumption that we can't have a file with a slash, we'll want to ensure that any addons that are created in the future are tested for that. (Yeah, I know, I'm nothing but trouble today)